### ISSUE TITLE
Package member suggestions does't work if package name is in same line as keyword

### ISSUE THREAD

**Issue by MadeManMax** [zabel-xyz/plsql-language#48]
Hi @zabel-xyz,
I've noticed that package member suggestions does't work if package name is in same line as keyword

This one doesn't work:
![obraz](https://user-images.githubusercontent.com/8720905/40721264-462ef29a-6419-11e8-8e5f-c4be0003810a.png)

This one works:
![obraz](https://user-images.githubusercontent.com/8720905/40721220-2e18b5b0-6419-11e8-9089-e96a9d84e286.png)

MyPackage1 package body
```sql
create or replace package body
MyPackage1
as

function get_myValue(param1 in varchar2)
    return varchar2
  is
  begin
    MyPackage12. -- here works
    RETURN MyPackage12. -- here doesn't work
    return param1 || ' TEST';
  end get_myValue;
end MyPackage1;
```

MyPackage12 package spec
```sql
create or replace package
MyPackage12
as
function get_myValue1111111111111111111
(param1 in varchar2
) return varchar2;


function get_myValue(param1 in varchar2)
    return varchar2;
end MyPackage12;
```

MyPackage12 package body
```SQL
create or replace package body
MyPackage12
as
function get_myValue1111111111111111111
(param1 in varchar2
) return varchar2;

function get_myValue(param1 in varchar2)
    return varchar2
  is
    x2 VARCHAR2(2000);
    x3 VARCHAR2(2000);
  begin
    if 1 =0 then
        null;
    end if;

    x2 := MyPackage2.get_myValue2(param1 => param1);
    x3 := MyPackage3.get_myValue3(param1 => param1);
    return param1 || ' TEST';
  end get_myValue;

end MyPackage12;
```

**Comment by zabel-xyz** [comment]
Fix in version 1.3.4

**Comment by MadeManMax** [comment]
Thank you
